packages/data/addon/adapters/facts/elide.ts

```diff
@@ -78,6 +78,10 @@ export default class ElideFactsAdapter extends EmberObject implements NaviFactAd
         return `${fieldStr}=ge=(${filterVals[0]});${fieldStr}=le=(${filterVals[1]})`;
       }
 
+      if (operator === 'nbet') {
+        return `${fieldStr}=le=(${filterVals[0]});${fieldStr}=ge=(${filterVals[1]})`;
```

Comment:
I think this should be `lt` and `gt`. I believe no rows that match values within the supplied interval should be included

Reply:
Oops, yeah what Jon said